rpa-builder/6.2/modules/ROOT/pages/toolbox-system-write-bot-message.adoc

```diff
@@ -0,0 +1,26 @@
+= Write Bot Message 
+
+The Write Bot Message Action Step enables you to enrich the run result message the RPA Bot generates after each run. This message shows in the run results page of RPA Manager. 
+
+== Properties 
+
+* *Bot message* 
++
+The text to write in the run result message. 
```

Comment:
There seems to be a "run result" message, so I'd choose to bold it or make it clear that this is a feature.